src/optimize.jl

```diff
@@ -3,26 +3,25 @@ using OptiMimi
 include("lib/readconfig.jl")
 include("lib/datastore.jl")
 
-config = readconfig("../configs/standard-1year.yml")
+config = readconfig("../configs/standard-60year-colorado.yml")
 
 suffix = getsuffix()
 
-include("optimization.jl")
+if config["cst"]=="true"
+    include("optimization_cst.jl")
+else
+    include ("optimization.jl")
+end
 
-using MathProgBase
+    
+    using MathProgBase
 @time sol = linprog(-house.f, house.A, '<', house.b, house.lowers, house.uppers)
 
-analysis = nothing
 
-if analysis == :shadowcost
-    varlens = varlengths(m, house.constcomps, house.constraints)
-    lambdas = sol.attrs[:lambda][sum(varlens[1:2])+1:sum(varlens[1:3])]
-    lambdas = reshape(lambdas, (3109, 2))
-    df = convert(DataFrame, lambdas)
-    df[:fips] = map(x -> parse(Int64, x), masterregions[:fips])
-    writetable("../results/shadowprice-1e6.csv", df)
-    usmap(DataFrame(fips=df[:fips], value=df[:x1]))
-elseif analysis == :debug
+
+debug = true #false true
+
+if debug
```

Comment:
Change this back to the `if analysis == ...` structure, since it's more general.